nvsvocprez/utils.py

```diff
@@ -250,4 +253,92 @@ def get_alt_profiles() -> Dict:
         return altprof_data_by_url
     except requests.RequestException as exc: 
         logging.error("Failed to retrieve alternate profile information from %s.\n%s", url, exc)
-        return {}   # Return blank list to avoid internal server error.
\ No newline at end of file
+        return {}   # Return blank list to avoid internal server error.
+    
+    
+    
+def get_alt_profile_objects(collection: Dict, alt_profiles: Dict) -> Dict:
```

Comment:
I like this method for convenience. From what I've seen already, mediatypes and default_mediatypes can vary. Maybe languages too later down the line. Would be good to make those 4 as optional params to this method, where the default value is what you have currently